Client/src/components/common/Input/index.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
-import './index.css';
 
-const Input = props =>
-  <input  {...props}  className='input' />;
-  
-export default Input;
+const Input = props => {
+  const {name, placeholder, id, onChange, inputClassName} = props
+  return (
+    <input name={name} placeholder={placeholder} id={id} onChange={onChange} className={inputClassName} />
```

Comment:
```suggestion
      const { inputClassName, ...resetProps } = 
    <input {...props} className={inputClassName} />
```

Reply:
I don't think this is a good idea 
because it will pass all the props in the input as a property and their values 
and some of them are not useful or in the right place